doc/whats_new.rst

```diff
@@ -19,6 +19,11 @@ New features
 Enhancements
 ............
 
+   - Added ``classes_`` attribute to :class:`model_selection.GridSearchCV`
+     that matches the ``classes_`` attribute of ``best_estimator_``. (`#7661
+     <https://github.com/scikit-learn/scikit-learn/pull/7661>`_) by `Alyssa
+     Batula`_ and `Dylan Werner-Meier`_.
```

Comment:
You need to add your names to the bottom of the file for the links to work. I'll do that.